.github/workflows/tests.yml

```diff
@@ -0,0 +1,46 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  schedule: 
+    - cron: '30 6 * * *'
+
+concurrency:
+  # Concurrency group that uses the workflow name and PR number if available
+  # or commit SHA as a fallback. If a new build is triggered under that
+  # concurrency group while a previous build is running it will be canceled.
+  # Repeated pushes to a PR will cancel all previous builds, while multiple
+  # merges to main will not cancel.
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  build_recipe:
+    name: Build conda recipe (${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ["3.9"]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: mamba-org/provision-with-micromamba@main
+        with:
+          environment-name: build
+          environment-file: false
+          channels: conda-forge
+          extra-specs: |
+            python=${{ matrix.python-version }}
+            conda-build
+            conda-libmamba-solver
+      - name: Build recipe
+        shell: bash -el {0}
+        env:
+          CONDA_SOLVER: libmamba
```

Comment:
You can use the new solver with `conda-build` just by setting this line. It should be equivalent to `mambabuild`.